bitnami/jaeger/templates/query/deployment.yaml

```diff
@@ -0,0 +1,192 @@
+      {{- end }}
+    spec:
+      {{- if .Values.query.schedulerName }}
+      schedulerName: {{ .Values.query.schedulerName }}
+      {{- end }}
+      priorityClassName: {{ .Values.query.priorityClassName | quote }}
+      {{- if .Values.query.affinity }}
+      affinity: {{- include "common.tplvalues.render" (dict "value" .Values.query.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.query.podAffinityPreset "component" "query" "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.query.podAntiAffinityPreset "component" "query" "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.query.nodeAffinityPreset.type "key" .Values.query.nodeAffinityPreset.key "values" .Values.query.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.query.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.query.podSecurityContext "enabled" | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.query.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.query.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.query.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.query.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.query.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.query.topologySpreadConstraints "context" $) | nindent 8 }}
+      {{- end }}
+      initContainers:
+      - name: jaeger-cassandra-schema-grabber
+        image: {{ include "jaeger.image" . }}
+        command: [ "cp", "-T", "-r", "/opt/bitnami/jaeger/cassandra-schema", "/cassandra-schema/" ]
+        volumeMounts:
+          - name: cassandra-schema
+            mountPath: "/cassandra-schema"
+      - name: jaeger-cassandra-schema-exec
+        image: {{ .Values.cassandra.image.registry }}/{{ .Values.cassandra.image.repository }}:{{ .Values.cassandra.image.tag }}
+        command:
+          - /bin/bash
+        args:
+          - -ec
+          - |
+            #!/bin/bash
+
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+
+            . /opt/bitnami/scripts/libos.sh
+
+            check_cassandra_connection() {
+                echo "SELECT 1" | cqlsh -u $CASSANDRA_USERNAME -p $CASSANDRA_PASSWORD -e "QUIT"
+            }
+
+            info "Connecting to the Cassandra instance $CQLSH_HOST:$CQLSH_PORT"
+            info "Take it easy. Could be authentication errors since the cassandra pod startup until the password is set"
+            if ! retry_while "check_cassandra_connection"; then
+            error "Could not connect to the database server"
```

Comment:
Indent this

Reply:
Fixed within the improvements of [a896c22](https://github.com/bitnami/charts/pull/13480/commits/a896c223d6f2e265ad494dd42b6553552d0a89b0)